src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -56,14 +60,12 @@ const NavSearchBar = () => {
     );
 
     const roomsResults =
```

Comment:
`roomsResults: Room[] =`

Reply:
I believe the intent of your comment here is to then be able to remove the `roomsResults as Room[]` below. In which case, I agree.

Reply:
Looking at this, it is complicated because `venue?.rooms?` is `Room[] | AnyRoom[]`, which is probably why i did the `as` in the first place.

I'll add a `@debt` comment, but probably won't try and change it in this PR.

I believe once we delete all of the old stuff in https://github.com/sparkletown/sparkle/pull/833 we should then be able to clean this up properly.